rbuild.ini

```diff
@@ -29,3 +29,12 @@ define =
     CMAKE_CXX_COMPILER_LAUNCHER=${deps_dir}/bin/ccache
     MIGRAPHX_ENABLE_CPU=On
     BUILD_DEV=On
+
+[cibuild]
+cxx = ${rocm_path}/llvm/bin/clang++
```

Comment:
I guess based on Paul's comment we'll need to add the ignore for rocm-cmake here as well

Reply:
No it was either ignore or add another session/profile from my interpretation.